modules/prometheus-service-monitor.adoc

```diff
@@ -8,6 +8,11 @@
 [role="_abstract"]
 If you are using the Prometheus Operator, you can use a service monitor to scrape the metrics from {rh-rhacs-first}.
 
+[IMPORTANT]
+====
+Prometheus operator monitoring is enabled by default for OpenShift 4 – no additional configuration is needed.
```

Comment:
What Prometheus operator? I think this is too unspecific.

Reply:
I added this warning based on abstract. should we link to https://github.com/prometheus-operator/prometheus-operator

Reply:
No, I would rather link to https://docs.openshift.com/container-platform/4.13/monitoring/monitoring-overview.html.

Reply:
Then, maybe we should mark this section as deprecated?

Reply:
It's not deprecated though, it's part of the "manual" Prometheus setup which is still supported. Just not the default or recommended use case.

Reply:
But I think we can remove the `IMPORTANT` message from this section, as "Prometheus operator" in this case typically does not refer to the OpenShift Prometheus operator but a custom one.

Reply:
But if somebody have used this to monitor with OpenShift monitoring then they might look at this section as they looked before so having a note is in my opinion good hint that it's not recommended and in fact required.